game/src/main/scala/io/aigar/game/Grid.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace._
+import java.util.Random
+
+object Grid {
+  final val WidthPerPlayer = 100
+  final val HeightPerPlayer = 100
+}
+class Grid(val width: Int, val height: Int) {
+  val random = new Random()
```

Comment:
> This constructor sets the seed of the random number generator to a value very likely to be distinct from any other invocation of this constructor.

[docs](https://docs.oracle.com/javase/7/docs/api/java/util/Random.html#Random())

It is seeded. If we were to manually seed it, we'd get the same game every time we would run it. I think it's valuable to leave it "unpredictable", just in case the game needs to be reset.